tests/integration/src/test/resources/org/mule/properties/attachment-transformer-test-case.xml

```diff
@@ -95,7 +95,7 @@
         <expression-transformer expression="#[message.outboundAttachments['attach']]"/>
         <expression-transformer expression="#[groovy:message.getPayload().getInputStream()]"/>
         <object-to-string-transformer/>
-        <test:assert expression="#[groovy:message.getPayloadAsString().equals('some text')]"/>
```

Comment:
What would be the way to get the payload as String now for objects that are not string?